cfme/tests/webui/test_expression_editor.py

```diff
@@ -0,0 +1,43 @@
+import fauxfactory
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+
+class TestDeleteExpressions(object):
+    filter_value = 'beautifulpotato'
+    filt = "Cloud Volume : Name"
+    collection = "volumes"  # any collection with advanced search
+    destination = "All"
+
+    @pytest.mark.meta(automates=[BZ(1720216)])
+    @pytest.mark.parametrize("operator", ["not", "and", "or"])
+    @pytest.mark.uncollectif(lambda operator, appliance:
+                             operator == "not" and appliance.version < "5.11")
+    def test_delete_operator(self, appliance, operator):
+        """
+        Polarion:
+            assignee: anikifor
+            casecomponent: WebUI
+            caseimportance: medium
+            initialEstimate: 1/30h
+        """
+        view = navigate_to(
+            getattr(
+                appliance.collections,
+                self.collection),
+            self.destination)
+        name = fauxfactory.gen_alphanumeric()
+        view.entities.search.save_filter(
+            f'fill_field({self.filt}, =, {self.filter_value});'
+            f'select_expression_text;'
+            f'click_{operator};'
+            f'{f"fill_field({self.filt}, =, {self.filter_value});" if operator != "not" else ""}'
+            f'select_expression_text;'
+            f'click_remove;',
+            name)
+        editor = view.search.advanced_search_form.search_exp_editor
+        assert editor.expression_text == f'{self.filt} = "{self.filter_value}"'
+        view.entities.search.delete_filter()
```

Comment:
optional:
If remove filter; I would add one more check for `expression_text`.

Reply:
Which check would you expect here, if expression_text is empty? Delete_filter is just cleanup, not essential for actual tests. I checked that even if I remove this statement, or put some expression in manually after delete_filter, next tests work without a problem - it automatically removes leftover statement.